pkgs/desktops/deepin/default.nix

```diff
@@ -25,6 +34,15 @@ let
     deepin-editor = callPackage ./apps/deepin-editor { };
     deepin-terminal = callPackage ./apps/deepin-terminal { };
 
+    #### Go Packages
+    go-lib = callPackage ./go-package/go-lib { inherit replaceAll; };
```

Comment:
Is it acceptable to place functions like this? @SuperSandro2000

Reply:
I don't think it is fully correct but we can fix that when more things are accumulated.